fitbook1/src/components/centercontent/createpost/index.ts

```diff
@@ -3,5 +3,21 @@ import { FaSmile } from "../../../assets/icons/icons";
 import PrimaryButton from "../../../components/buttons/PrimaryButton";
 import { useState } from "react";
 import Picker from "emoji-picker-react";
+import { createPost } from "../../../util/api/createPost";
+import { useAppSelector } from "../../../app/hooks";
+import { useAppDispatch } from "../../../app/hooks";
+import { postsActions } from "../../../app/features/posts/postSlice";
+import { showToast } from "../../../util/toasts/showToast";
 
```

Comment:
suggestion: you could do exports from an index file

Reply:
Done